keyboards/pizzakeyboards/slice65/info.json

```diff
@@ -0,0 +1,104 @@
+                {"label": "]", "matrix": [2, 6], "x": 12.5, "y": 1},
+                {"label": "\\", "matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
+                {"label": "End", "matrix": [2, 7], "x": 15, "y": 1},
+                {"label": "Capslock", "matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "A", "matrix": [5, 0], "x": 1.75, "y": 2},
+                {"label": "S", "matrix": [4, 1], "x": 2.75, "y": 2},
+                {"label": "D", "matrix": [5, 1], "x": 3.75, "y": 2},
+                {"label": "F", "matrix": [4, 2], "x": 4.75, "y": 2},
+                {"label": "G", "matrix": [5, 2], "x": 5.75, "y": 2},
+                {"label": "H", "matrix": [4, 3], "x": 6.75, "y": 2},
+                {"label": "J", "matrix": [5, 3], "x": 7.75, "y": 2},
+                {"label": "K", "matrix": [4, 4], "x": 8.75, "y": 2},
+                {"label": "L", "matrix": [5, 4], "x": 9.75, "y": 2},
+                {"label": ";", "matrix": [4, 5], "x": 10.75, "y": 2},
+                {"label": "'", "matrix": [5, 5], "x": 11.75, "y": 2},
+                {"label": "Enter", "matrix": [5, 6], "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "PgUp", "matrix": [4, 7], "x": 15, "y": 2},
+                {"label": "Shift", "matrix": [6, 0], "x": 0, "y": 3, "w": 2.25},
+                {"label": "Z", "matrix": [6, 1], "x": 2.25, "y": 3},
+                {"label": "X", "matrix": [7, 1], "x": 3.25, "y": 3},
+                {"label": "C", "matrix": [6, 2], "x": 4.25, "y": 3},
+                {"label": "V", "matrix": [7, 2], "x": 5.25, "y": 3},
+                {"label": "B", "matrix": [6, 3], "x": 6.25, "y": 3},
+                {"label": "N", "matrix": [7, 3], "x": 7.25, "y": 3},
+                {"label": "M", "matrix": [6, 4], "x": 8.25, "y": 3},
+                {"label": ",", "matrix": [7, 4], "x": 9.25, "y": 3},
+                {"label": ".", "matrix": [6, 5], "x": 10.25, "y": 3},
+                {"label": "/", "matrix": [7, 5], "x": 11.25, "y": 3},
+                {"label": "Shift", "matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "Up", "matrix": [7, 6], "x": 14, "y": 3},
+                {"label": "PgDn", "matrix": [6, 7], "x": 15, "y": 3},
+                {"label": "Ctrl", "matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
+                {"label": "Win", "matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "Alt", "matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "Space", "matrix": [8, 3], "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "Alt", "matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
+                {"label": "Ctrl", "matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "Left", "matrix": [8, 6], "x": 13, "y": 4},
+                {"label": "Down", "matrix": [9, 6], "x": 14, "y": 4},
+                {"label": "Right", "matrix": [8, 7], "x": 15, "y": 4}
+            ]
+        }
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
`qmk format-json -i` removes the last newline character.  Should I be adding it back in after running it?

Reply:
Unfortunately yes

Reply:
This has been fixed in develop, just FYI.